thinc/schedules.py

```diff
@@ -73,50 +96,54 @@ def slanted_triangular(
     ratio: int = 32,
     decay: float = 1.0,
     t: float = 0.0,
-) -> Iterable[float]:
+) -> ScheduleCallable:
     """Yield an infinite series of values according to Howard and Ruder's
     "slanted triangular learning rate" schedule.
     """
     cut = int(num_steps * cut_frac)
-    while True:
-        t += 1
+
+    def callback(*, step: int, **kwargs) -> float:
+        t = step + 1
```

Comment:
As with `decaying`, this should be something like `t_new = t + step + 1` to preserve the old functionality.